tests/FactoryTest.php

```diff
@@ -52,11 +52,11 @@ public function alphabetsProvider(): array
      *
      * @dataProvider alphabetsProvider
      */
-    public function testSetAlphabet(string $alphabet): void
+    public function testSetAlphabet($alphabet): void
```

Comment:
To be consistency and coding style is set on `.php_cs`, the method has used with the `@param` doc comment.

And it's unnecessary to use the native `string` type for `$alphabet` argument.

Reply:
Please explain "unnecessary"